### PR TITLE
Always use error.Message

### PR DESCRIPTION
**Pull request by johnsudaar**

### CODE REVIEW

File: hook.go [hunk 1]
```diff
@@ -65,18 +65,17 @@ func (h hook) Fire(entry *logrus.Entry) error {
...
+	msg := entry.Message
+	if entry.Data["msg"] != nil {
+		msg = fmt.Sprintf("%s - %v", msg, entry.Data["msg"])
+	}
+
 	if entry.Data["error"] != nil {
 		err = entry.Data["error"].(error)
 
 		errorTxt := new(bytes.Buffer)
 		errorTxt.WriteString(err.Error())
```
Comment (Soulou) [inline hook.go]: Here the error will be `<nil> - something`, don't you want to hide completely the `<nil>` part please?

Comment (johnsudaar) [reply hook.go]: No

Comment (johnsudaar) [reply hook.go]: https://github.com/Scalingo/logrus-rollbar/pull/9/files#diff-6824f41d082d8d1e355aae9838a40779R73

Comment (Soulou) [reply hook.go]: my bad, I misread

**Pull request merged**